src/MyFailedClass4.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+class MyFailedClass4
+{
+	public function test(): string
+	{
+		$functionsCookies = $_COOKIE;
```

Comment:
:no_entry_sign: **Error**: Due to server-side caching, server-side based client related logic might not work. We recommend implementing client side logic in JavaScript instead (*WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___COOKIE*).